examples/vic-gov-au/nuxt.config.js

```diff
@@ -123,6 +123,9 @@ export default {
       username: process.env.CONTENT_API_AUTH_USER,
       password: process.env.CONTENT_API_AUTH_PASS
     },
+    oauth: {
+      clientId: process.env.CONTENT_API_CLIENT_ID
```

Comment:
The reason I added it in this way was to get the test environments set up. While we could access the env variable directly in the preview/module.js, accessing it through the nuxt config does feel like the right way to do it, considering most other env variables are accessed this way. I was also wondering about potential future changes, if scopes need to expand and need to be set unique for some sites, this could be a place where those options are set as well.
But I'm happy to change if it's going to make it easier for deployment.

Reply:
Thanks @alan-cole I get what you mean. We can also move the client id setting to preview module just like this: https://github.com/dpc-sdp/ripple/blob/master/examples/vic-gov-au/nuxt.config.js#L137 , I feel that will make more sense.
But for now I would like just not add that config until we need to share this client id with other code. Just for reduce the maintenance work.

Reply:
Reverted to previous implementation. Accesses env variable directly.